src/components/search-query.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch';
+import PropTypes from 'prop-types';
+
+import { RefinementList, InstantSearch, SearchBox, Hits, ClearRefinements, Pagination } from 'react-instantsearch-dom';
+import webComponent from '../web-component';
+
+const searchClient = algoliasearch('H897LKXYG1', '50a4f124e0d934cac92e79ece376316a');
+
+const hitComponent = ({ hit }) => (
+  <div>
+    { hit.last_name ?
+      <div><strong>Contributor:</strong> {hit.first_name} {hit.last_name}</div> : ''
+    }
+    <div><strong>Election: </strong>
+      <a href={`/election/${hit.election_location.toLowerCase()}/${hit.election_date}`} >
+        {hit.election_title}
+      </a>
+    </div>
+    { hit.title ?
+      <div><strong>Ballot Measure: </strong>
+        <a href={`/referendum/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.slug}`} >
+          {hit.title}
+        </a>
+      </div> : ''}
+    { hit.name ?
+      <div><strong>Candidate: </strong>
+        <a href={`/candidate/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.candidate_slug}`} >
+          {hit.name}
+        </a>
+      </div>
+     : ''
+    }
+    { hit.office_title ?
+      <div>
+        <strong>Office: </strong>
+        <a href={`/office/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.office_slug}`} >
+          {hit.office_title}
+        </a>
+      </div> : ''
+    }
+    { hit.amount ? <div><strong>Amount:</strong> ${hit.amount}</div> : ''}
+    <strong>Election date:</strong> {hit.election_date}<br />
+    <strong>Location:</strong> {hit.election_location}<br />
```

Comment:
```suggestion
    
    {hit.amount ? <div><strong>Amount: </strong>${hit.amount}</div> : ''}

    <div><strong>Election date: </strong>{hit.election_date}</div>
    <div><strong>Location: </strong>{hit.election_location}</div>
```

Reply:
Why is this not:
     { hit.amount ?                                                                 
       <div>                                                                        
         <strong>Amount:</strong> ${hit.amount}                                     
       </div>                                                                       
       : ''}

Reply:
Good question - I left this one because it fit on one line. Most of my suggestions are for helping readability of the code overall, and I felt that this one didn't need the multi-line pattern to help with reading it. But, I don't think the multi-line pattern is necessarily wrong for this one, so either way I think it is fine!